### PR TITLE
Small corrections + new toolbox functions.

### PR DESCRIPTION
**Pull request by oualib**
- Small typo corrections
 - New tool box function to help on the issue #418

### CODE REVIEW

File: verticapy/toolbox.py [hunk 1]
```diff
@@ -1610,3 +1610,300 @@ def __round__(self, x):
 
     def category(self):
         return self.category_
+
+#
+#
+# Tools to merge similar names/categories together.
+#
+# ---#
+def erase_prefix_in_name(name: str, prefix: list = []):
```
Comment (mail4umar) [inline verticapy/toolbox.py:1620]: Currently if we have two prefixes but the sequence inside the name does not match the sequence of prefix list then one of the prefix can remain. if that is intentional, then it is fine.

For example:
erase_prefix_in_name(name="pre1pre2waro",prefix=["pre2","pre1"])
OUPUT: 
'pre2waro'

UPDAtE: now that i re-read the details. i think you intend this to happen. SO PLEASE IGNORE IF THAT IS THE CASE

File: verticapy/toolbox.py [hunk 2]
```diff
@@ -1610,3 +1610,300 @@ def __round__(self, x):
...
+-------------------------------------------------------------------------------
+Excludes the input lists of suffixes and prefixes from the input name and 
+returns it. When there is a match, the other elements of the list are ignored.
+
+Parameters
+---------- 
+name: str
+    Input name.
+suffix: list, optional
+    List of suffixes.
```
Comment (mail4umar) [inline verticapy/toolbox.py:1719]: @oualib  Why do we want only 1 suffix to be removed and rest to be ignored?

File: verticapy/toolbox.py [hunk 3]
```diff
@@ -1610,3 +1610,300 @@ def __round__(self, x):
...
+    order: list = ["p", "s", "w"]
+):
+    """
+-------------------------------------------------------------------------------
+Excludes the input lists of suffixes, prefixes and words from the input name 
+and returns it.
+
+Parameters
+---------- 
+name1: str, optional
```
Comment (mail4umar) [inline verticapy/toolbox.py:1764]: i dont think mae1 name2 are optional as the function below does erase does not have the name as optional,

File: verticapy/toolbox.py [hunk 4]
```diff
@@ -1610,3 +1610,300 @@ def __round__(self, x):
...
+    order: list = ["p", "s", "w"]
+):
+    """
+-------------------------------------------------------------------------------
+Excludes the input lists of suffixes, prefixes and words from the input name 
+and looks if it belongs to a specific group.
+
+Parameters
+---------- 
+name: str, optional
```
Comment (mail4umar) [inline verticapy/toolbox.py:1813]: i dont think they are optional

File: verticapy/toolbox.py [hunk 5]
```diff
@@ -1610,3 +1610,300 @@ def __round__(self, x):
...
+    Input Name.
+group: list, optional
```
Comment (mail4umar) [inline verticapy/toolbox.py:1815]: not optional. used in for loop

**Review by mail4umar — CHANGES_REQUESTED** [review]

File: verticapy/toolbox.py (hunk 2 above)
Comment (oualib) [reply verticapy/toolbox.py:1719]: I wanted to only test one. these functions are used to only simplify one future function.

**Review by mail4umar — APPROVED** [review]

**Pull request merged**